computerGroups.go

```diff
@@ -0,0 +1,122 @@
+package jamf
+
+import "fmt"
+
+const uriComputerGroups = "/JSSResource/computergroups"
+
+type ComputerGroupsResponse struct {
+	Count   int                         `xml:"size"`
+	Results []ComputerGroupListResponse `xml:"computer_group"`
+}
+
+type ComputerGroupListResponse struct {
+	ID      int    `xml:"id,omitempty"`
+	Name    string `xml:"name,omitempty"`
+	IsSmart bool   `xml:"is_smart,omitempty"`
+}
+
+type ComputerGroup struct {
+	ID            int                      `xml:"id"`
+	Name          string                   `xml:"name"`
+	IsSmart       bool                     `xml:"is_smart"`
+	Site          Site                     `xml:"site"`
+	Criteria      []ComputerGroupCriterion `xml:"criteria>criterion"`
+	CriteriaCount int                      `xml:"criteria>size"`
```

Comment:
`Size` is return for the API. 

I thought it would be easier to understand if the field name was `CriteriaSize`